pkg/trace/obfuscate/json.go

```diff
@@ -69,7 +69,7 @@ func newJSONObfuscator(cfg *config.JSONObfuscationConfig, o *Obfuscator) *jsonOb
 
 func sqlObfuscationTransformer(o *Obfuscator) func(string) string {
 	return func(s string) string {
-		result, err := o.ObfuscateSQLString(s)
+		result, err := o.ObfuscateSQLString(s, nil)
```

Comment:
Nit: should you consider a `ObfuscateSQLStringWithOptions(string, SQLObfuscationConfig)`?
This would leave you with `o.ObfuscateSQLString(string)` (that calls the WithOptions without options).

Reply:
I really like this pattern, thanks! Updated: https://github.com/DataDog/datadog-agent/pull/8556/commits/cedf48d1708f11ce9213e53b7c5cbc90d7f21705